docs/state-store-overview.md

```diff
@@ -34,6 +42,38 @@ Streaming state store has distinguished workload characteristics.
 
 This leads to the design of Hummock, the cloud-native KV-based streaming state store. We’ll explain concepts like “epoch”, “key space” and “barrier” in the following chapters.
 
+## Cell-based Relational Table
+[source code](https://github.com/singularity-data/risingwave/blob/main/src/storage/src/table/state_table.rs)
+
+In this part, we will introduce how stateful executors interact with KV state store through the relational table layer.
+
+Relational table layer consists of State Table, Mem Table and Cell-based Table. The State Table provides the table operations by these APIs: `get_row`, `scan`, `insert_row`, `delete_row` and `update_row`, which are the read and write  interfaces for executors. The Mem Table
```

Comment:
redundant space and line break

Reply:
Shall we elaborate on the meaning of "cell-base" in this section?

Reply:
Just make line breaks for markdown.

Reply:
> Shall we elaborate on the meaning of "cell-base" in this section?

Probably not necessary